readthedocs/settings/base.py

```diff
@@ -145,6 +145,7 @@ def INSTALLED_APPS(self):  # noqa
 
             # our apps
             'readthedocs.projects',
+            'readthedocs.organizations',
```

Comment:
I assume that installing this app won't have any effect in .org until we register the URLs. It may be better to that time before adding it to the INSTALLED_APPS.

Reply:
It doesn't have any effect on .org, but we need it so django can manage the models. We can install the app only on .com maybe.